infrastructure/storage/src/lmdb.rs

```diff
@@ -257,13 +290,22 @@ mod test {
         let val = store.get_raw(b"England").unwrap().unwrap();
         assert_eq!(str::from_utf8(&val).unwrap(), "rose");
         // Clean up
-        assert!(fs::remove_dir_all("./tests/test_default").is_ok());
+        assert!(store.delete_db_from_scope().is_ok());
+        let _no_val = fs::remove_dir_all(test_dir);
+        if std::fs::metadata(test_dir).is_ok() {
+            println!("Database file handles not released, still open in {:?}!", test_dir);
```

Comment:
Should ```println```not be removed from all the tests?

Reply:
That is the only way to get some meaningful feedback during run-time if clean up dos not work, see below:

```
Testing started at 07:31 ...
Database file handles not released, still open in "./tests/test_default"!

thread 'lmdb::test::writes_to_default_db' panicked at 'assertion failed: fs::remove_dir_all(test_dir).is_ok()', infrastructure\storage\src\lmdb.rs:297:13
note: Run with `RUST_BACKTRACE=1` environment variable to display a backtrace.
```